sdk/keyvault/azure-keyvault-administration/azure/keyvault/administration/_enums.py

```diff
@@ -0,0 +1,77 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+from enum import Enum
+
+
+class KeyVaultRoleScope(str, Enum):
+    """Collection of well known role scopes. This list is not exhaustive"""
+
+    global_value = "/"  #: use this if you want role assignments to apply to everything on the resource
```

Comment:
Is the "_value" suffix common in Python? In .NET we just use "Global" and "Keys". If so, cool.

Reply:
"global" is a keyword. I forget how we ended up with _value here; looking at it again I like "global_scope" and "keys", what do you think @mccoyp?

Reply:
If we decide to go with a `_scope` suffix, I'd suggest we do it for both.

Reply:
Thinking about how a user would use the enum and because "Scope" is already in the class name, I think "global" and "keys" would make sense, without a suffix. I also wondered about whether these should be in caps -- I kept them in snake case for consistency with other libraries, but I was under the impression that enum values should be in caps

Reply:
I'm happy with caps; consistency is less important to me than making the better choice here.